src/fetchcode/package.py

```diff
@@ -314,16 +312,40 @@ def get_rubygems_data_from_purl(purl):
     api_url = f"https://rubygems.org/api/v1/gems/{name}.json"
     response = get_response(api_url)
     declared_license = response.get("licenses") or None
+    version = response.get("version")
+    version_purl = PackageURL(
```

Comment:
store type as a variable and use it again here https://github.com/nexB/fetchcode/pull/84/files#diff-0e634dc0ba62fc770fb30d4dbc821e8a87a4b5aafd38b27fb1ab26aafb098933R340

Reply:
Please use different variable names for both purls here and there https://github.com/nexB/fetchcode/pull/84/files#diff-0e634dc0ba62fc770fb30d4dbc821e8a87a4b5aafd38b27fb1ab26aafb098933R340 to improve readability.